exercises/d2e3.js

```diff
@@ -17,15 +17,20 @@ const alarms = [
 ]
 
 app.get('/alarms', (req, res) => {
-  /**
-   * Responds with all the alarms as a JSON response
-   */
+
+res.json(alarms)
```

Comment:
Nit: Again, watch your spacing here to keep it readable 😄 I've also noticed that you usually terminate your statements with a semicolon `;`, which is best practice. Many people choose not to do this in JavaScript because of Automatic Semicolon Insertion, though it can introduce bugs.

It's important to be consistent in your application, though, so add it here, given that you've started!
```suggestion
  res.json(alarms);
```